runtime/src/execute_cost_table.rs

```diff
@@ -106,6 +106,17 @@ impl ExecuteCostTable {
             }
         }
 
+        // Logs events of unusual input `value` or `aggregated.ema`
+        const MAX_COST: u64 = 200_000;
+        if let Some(aggregated) = self.table.get(key) {
+            if value > MAX_COST || aggregated.ema > MAX_COST as f64 {
+                debug!(
```

Comment:
kinda torn here as I want to see these in metrics (though probably aggregated), but not spam the log :thinking:

Reply:
tough call here. It is datapoint per instruction, so there are tons of data. If aggregate, then those triggering events are likely be folded into averaged data. I'm thinking to just capture those unusual events (either input data or calc results are too big) to sound alarm, not meant to get stats. But I can go both ways.